src/cmd-test-coreos-installer

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+# Automate an end-to-end run of coreos-installer with the metal image, which then
+# boots and writes a success message to a virtio-serial port, which we read on the host.
+set -euo pipefail
+
+run_tmp_webserver() {
```

Comment:
Ohh and I forgot about https://github.com/coreos/mantle/pull/1141 will change to use that